src/test.js

```diff
@@ -24,6 +24,7 @@ import ProcessingQueue from "./core/processing-queue";
 import TestReport from "./reports/test";
 
 let focused = false;
+let someFocused = false;
```

Comment:
Alternatively:
```js
const FOCUSED_ONLY = "only";
const FOCUSED_SOME = "some";
const FOCUSED_NONE = "none";

let focused = FOCUSED_NONE;

function isFocused() {
  return [FOCUSED_ONLY, FOCUSED_SOME].includes(focused);
}
```
then (when needed) we can simply check `focused === FOCUSED_SOME` etc...